_pages/2024/shared-tasks/simultaneous.md

```diff
@@ -13,40 +13,172 @@ if you want to have a line break either put a blank line in between the text or
 
 ## Description
 
-Details coming soon!
+Simultaneous translation (also known as real-time or streaming translation) is the task of generating translations incrementally given partial input only.
+Simultaneous translation enables interesting applications such as automatic simultaneous interpretation or international conference translations.
+Simultaneous systems are typically evaluated with respect to quality and latency.
 
-<!-- Description the task, the languages, and the type of data -->
+There will be 2 tracks:
 
+- Speech-to-Text: simultaneously translating speech in source language into text in target language.
+- Speech-to-Speech: simultaneously translating speech in source language into speech in target language.
 
-## Data
+and three language directions:
+
+- English -> German
+- English -> Chinese
+- English -> Japanese
+- (One more language direction in consideration)
 
-Data will be released in January.
 
-<!-- Details description of the data and links to download -->
+## Data
+
+The data condition for this task is "constrained with pretrained models". The list of allowed data and pretrained models can be found [here](https://iwslt.org/2024/offline#training-data-and-data-conditions).
 
 
 ## Baselines
 
-<!-- Links to the baselines to be used (descriptions, publications and/or links to models, code) -->
+(Version 2023. New baselines will be provided later.)
+We provide a test-time [Wait-K](https://aclanthology.org/P19-1289/) speech-to-text baseline as follows:
+
+| Language Direction | Latency (seconds) | Quality (BLEU) | Offline Quality (BLEU) |
+| ------------------ | ----------------- | -------------- | ---------------------- |
+| En -> De           | 1.98              | 14.3           | 23.2                   |
+| En -> Zh           | 1.77              | 14.2           | 18.7                   |
+| En -> Ja           | 1.82              | 6.63           | 11.2                   |
+
+The instruction to reproduce the baseline results can be found [here](https://github.com/facebookresearch/fairseq/tree/iwslt2023/examples/simultaneous_translation).
 
 
 ## Submission
 
-<!-- Description of expected submission format and submission instructions -->
+### Format
+
+Participants can have multiple submissions,
+but can only have one submission for one track (`s2t`/`s2s`), one language (`de`/`zh`/`ja`).
+
+One submission should contain the following items:
+
+- **A built docker image file**, named `image.tar`.
+- The output logs of the system on **MuST-C v2.0 tst-COMMON** sets, named `results`.
```

Comment:
It is important to use v2 as the test set is slightly different from the first version.